WebInkEnhancement/explainer.md

```diff
@@ -69,7 +69,7 @@ As Pointer events gets delivered to an app, application continues rendering ink,
 const renderer = new InkRenderer();
 
 try {
-    let presenter = await navigator.ink.requestPresenter('pen-stroke-tip');
+    let presenter = await navigator.ink.requestPresenter('pen-stroke-tip', canvas);
     renderer.setPresenter(presenter);

```

Comment:
Can we add usage of the expectedImprovement property here? Maybe use that to set a bool (set same bool in the `except` block) that determines whether or not we need to fall back to the non-InkPresenter code path.

Reply:
Seems like just throwing an error to force fallback if the expectedImprovement value is too low is a simpler usage, let me know if you disagree.